scripts/Tools/case.setup

```diff
@@ -41,14 +41,17 @@ OR
 
     parser.add_argument("-t", "--test-mode", action="store_true",
                         help="Keeps the test script when the --clean argument is used")
+
     parser.add_argument("-r", "--reset", action="store_true",
                         help="Does a clean followed by setup")
 
+    parser.add_argument("--no-adjust-pio", action="store_true",
+                        help="Do NOT adjust pio settings for new pelayout")
```

Comment:
Hows this?
     help="Do NOT adjust pio settings for new pelayout."
                                  "By default if the pelayout is changed the pio layout will be adjusted."
                                  "This option overrides that feature and leaves the pio layout as currently set."